server/routes/missions.js

```diff
@@ -22,26 +22,28 @@ function authorizedUser(req, res, next) {
 
 // need to display user-specific missions + casefiles when user is logged in
 router.get('/api/missions', (req, res, next) => {
-  let files = {};
+  let missionsArray = [];
+
   // TODO where user_id === logged_in user (req.session.user)
   let user = 1; // temporary workaround
 
   Mission.forge().where({user_id: user}).query('orderBy', 'id', 'asc')
     .fetchAll({withRelated: ['casefile'], debug:true})
-    .then((mission) => {
+    .then((missions) => {
       // convert data to JSON
-      mission = mission.toJSON();
-      // loop over data to get mission and casefile names
-      for (var i = 0; i < mission.length; i++) {
-        // save to files object
-        if (mission[i].casefile && mission[i].casefile.name) {
-          files[mission[i].name] = mission[i].casefile.name;
-        } else {
-          files[mission[i].name] = "no casefile added";
+      missions = missions.toJSON();
+
+      // loop over data to get mission and casefile names & ids
+      for (var i = 0; i < missions.length; i++) {
```

Comment:
There is a lot of noise in this PR bc I went through and refactored a bunch to clean up the console warnings.

But I wanted to make sure I pointed this out @rallyjinx. I modified the `/api/missions` endpoint to return an array of mission objects. Previously we were returning it as an object with a list of key, value pairs where the key was the mission name and the value was the casefile. 

This reformatting allows us to pass more than just two data attributes about a Mission. 

Hopefully that makes sense.